intranet/static/js/eighth/schedule.js

```diff
@@ -175,5 +175,20 @@ $(function() {
         if (activities !== "" && !confirm("Are you sure you want to add the following activities without a sponsor?\n" + activities)) {
             e.preventDefault();
         }
+        $("tr.form-row:not(.hidden)").each(function(i, el) {
+          if($(this).hasClass("scheduled")){
+              var stickyCancel = "";
+              var scheduledField = $("td[data-field='scheduled'] input", el);
+              var stickyField = $("td[data-field='sticky'] input", el);
+              if (!scheduledField.prop("checked")) {
+                  // TODO: Implement check for sticky flag on activity.
+                  if(stickyField.prop("checked") || ((activityIsSticky == True) && !$("input[type='checkbox'].sticky").prop("checked"))) {
```

Comment:
```suggestion
                  if(stickyField.prop("checked") || activityIsSticky) {
```
I *think* this is the right condition. Regardless, your use of `True` is incorrect -- in Javascript it's `true`.